torch/_dynamo/variables/builder.py

```diff
@@ -727,6 +727,7 @@ def index_source(key):
             istype(value, (type, types.FunctionType))
             and skipfiles.check(value, allow_torch=True)
             and not inspect.getattr_static(value, "_torchdynamo_inline", False)
+            and not inspect.getattr_static(value, "__script_if_tracing_wrapper", False)
```

Comment:
What's going on here?

Reply:
This is to fix a latent bug, where we should check the function unpacked from ```torch.jit.script_if_tracing``` decorated function.